blocks/tabs/tabs.js

```diff
@@ -0,0 +1,132 @@
+    content.splice(1, 3, textWrapper);
+
+    const hrefs = [content[2].children[0].href, content[3].children[0].href];
+    const buttons = [
+      domEl('calcite-button', {
+        'icon-end': 'play-f',
+        href: hrefs[0],
+        appearance: 'solid',
+        alignment: 'center',
+        scale: 'm',
+        type: 'button',
+        width: 'auto',
+        kind: 'inverse',
+        color: 'inverse',
+        'calcite-hydrated': '',
+      }, content[2].textContent),
+      domEl('calcite-button', {
+        'icon-end': 'arrowRight',
+        href: hrefs[1],
+        appearance: 'outline',
+        alignment: 'center',
+        scale: 'm',
+        type: 'button',
+        width: 'auto',
+        kind: 'inverse',
+        'calcite-hydrated': '',
+      }, content[3].textContent),
+    ];
+
+    const buttonsWrapper = div({ class: 'buttons-wrapper' }, ...buttons);
+    content.splice(2, 2, buttonsWrapper);
+  });
+
+  const contents = tabContents.map((content, index) => div({ class: 'tab-content', id: `tab-content-${index}` }, ...content));
+  const titles = tabTitles.map((title, index) => li({ class: 'tab-title', id: `tab-title-${index}` }, button(...title)));
+
+  const arrowLeft = button(
+    {
+      class: 'arrow-button left',
+      'aria-hidden': 'true',
+    },
+    domEl('calcite-icon', {
+      icon: 'chevronLeft',
+      scale: 'm',
+      'calcite-hydrated': '',
+    }),
+  );
+
+  const arrowRight = button(
+    { class: 'arrow-button right' },
+    domEl('calcite-icon', {
+      icon: 'chevronRight',
+      scale: 'm',
+      'calcite-hydrated': '',
+    }),
+  );
+
+  let visibleTitleIdx = 0;
+  arrowLeft.addEventListener('click', () => {
+    const newVisibleTitleIdx = visibleTitleIdx - 1;
+    if (newVisibleTitleIdx < 0) return;
+    if (newVisibleTitleIdx === 0) arrowLeft.setAttribute('aria-hidden', 'true');
+    arrowRight.setAttribute('aria-hidden', 'false');
+
+    titles[visibleTitleIdx].classList.toggle('visible');
+    titles[newVisibleTitleIdx].classList.toggle('visible');
+
+    visibleTitleIdx = newVisibleTitleIdx;
+  });
+
+  arrowRight.addEventListener('click', () => {
+    const newVisibleTitleIdx = visibleTitleIdx + 1;
+    if (newVisibleTitleIdx >= titles.length) return;
+    if (newVisibleTitleIdx === titles.length - 1) arrowRight.setAttribute('aria-hidden', 'true');
+    arrowLeft.setAttribute('aria-hidden', 'false');
+
+    titles[visibleTitleIdx].classList.toggle('visible');
+    titles[newVisibleTitleIdx].classList.toggle('visible');
+
+    visibleTitleIdx = newVisibleTitleIdx;
+  });
+
+  const calciteTabs = div(
+    { class: 'tab-component' },
+    ul(
+      { class: 'tab-nav' },
+      arrowLeft,
+      ...titles,
+      arrowRight,
+    ),
+    ...contents,
+  );
+
+  contents[0].classList.toggle('selected');
+  titles[0].classList.toggle('selected');
+  titles[visibleTitleIdx].classList.toggle('visible');
+
+  titles.forEach((title, index) => {
+    title.addEventListener('click', () => {
```

Comment:
On the live site clicking also adds an anchor link: https://www.esri.com/en-us/capabilities/imagery-remote-sensing/capabilities/management#manage-imagery-and-raster-data

We should also mimic the behaviour where it scrolls to the element on page load (with the matching tab open)